language/evm/hardhat-examples/test/Greeter.test.js

```diff
@@ -1,7 +1,7 @@
 const { expect } = require("chai");
 const { ethers } = require("hardhat");
 
-describe("Greeter", function () {
+describe("Greeter (the Move contract)", function () {
```

Comment:
I merged some of the tests using the "make_test" pattern. Some tests were not merged because they are different to workaround the unsupported feature of "move-to-yul" for now.